conda/recipes/librmm/recipe.yaml

```diff
@@ -0,0 +1,105 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  head_rev: ${{ git.head_rev( "." )[:8] }}
+
+recipe:
+  name: librmm-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - cmake ${{ cmake_version }}
+      - ninja
+      - ${{ compiler("c") }}
+      - ${{ compiler("cxx") }}
+      - ${{ compiler("cuda") }}
+      - cuda-version =${{ cuda_version }}
+      - ${{ stdlib("c") }}
+
+  build:
+    script:
+      file: build.sh
+      secrets:
+        - AWS_ACCESS_KEY_ID
+        - AWS_SECRET_ACCESS_KEY
+        - AWS_SESSION_TOKEN
+
+      env:
+        CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER", default="sccache") }}
+        CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR", default="Ninja") }}
+        PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL", default="4") }}
+        SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET", default="") }}
+        SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT", default="") }}
+        SCCACHE_REGION: ${{ env.get("SCCACHE_REGION", default="") }}
+        SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL", default="") }}
+        SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS", default="") }}
+        SCCACHE_S3_KEY_PREFIX: librmm-${{ env.get("RAPIDS_CONDA_ARCH", default="") }}
```

Comment:
I don't think we want a default here. Otherwise we might get behavior that tries to read/write `librmm-` when the default `""` is used. That would be a bug.

Can we make _all_ of these env vars required? Certainly all the sccache-related ones should be mandatory imo, as they force our CI to be correct. For various reasons, such as our deep reliance on `gha-tools`, we don't expect recipes to be built outside of our CI images, so I am okay with having the recipes enforce correctness in CI.

Reply:
Yeah, it's helpful for local iteration to have defaults but I can remove them now that we're close to something polished.